### ISSUE TITLE
[frio] Screen Reader Accessibility Part 5: Move title to time tag

### ISSUE THREAD

**Issue by MrPetovan** [friendica/friendica#10979]
@pvagner

> Hello, I have found another... On the timeline each status or a comment includes a time like this.
> 
> ```
> <span class="time" title="" data-toggle="tooltip" data-original-title="Wed, 10 Nov 2021 03:42:23 +0100">
> 	<time class="dt-published" datetime="Wed, 10 Nov 2021 03:42:23 +0100">3 days ago (Received 1 day ago)</time>
> </span>
> ```
> 
> When this part is read with a screen reader the span is read first. It has no label, no role and has a title attribute. Out of the title so called accessible description is generated for that element. Then there is time with a relative time and no tooltip. It would be better if the relate time got read first and then the description. Would it be acceptable to move the title to the time tag and leave out the title attribute from the span element?

Source: https://github.com/friendica/friendica/issues/1743#issuecomment-967603239
